package.json

```
@@ -16,7 +16,7 @@
     "build-bigint-buffer": "node scripts/build-bigint-buffer.js",
     "postinstall": "node scripts/build-bigint-buffer.js",
     "start": "yarn typechain && nodemon -e ts,tsx,json,js,jsx --watch ./src --ignore ./dist --exec 'yarn dev'",
-    "build": "yarn run clean && yarn typechain && yarn run build:cjs & yarn run build:esm & yarn run build:types; wait",
+    "build": "yarn run clean && yarn typechain && concurrently --kill-others-on-fail --names \"cjs,esm,types\" --prefix-colors \"blue,magenta,green\" \"yarn run build:cjs\" \"yarn run build:esm\" \"yarn run build:types\"",
     "dev": "yarn run build:cjs & yarn run build:esm & yarn run build:types; wait",
```

How about something like this? btw, do we need the escaped quotes, or could we use `'` instead?

```suggestion
    "build": "yarn run clean && yarn typechain && yarn dev",
    "dev": "concurrently --kill-others-on-fail --names \"cjs,esm,types\" --prefix-colors \"blue,magenta,green\" \"yarn run build:cjs\" \"yarn run build:esm\" \"yarn run build:types\"",
```

---

https://github.com/across-protocol/sdk/pull/943/commits/5861a3894d38ad39953a1b1468ab364355c06386